packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/workspace-group.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import ToolbarIcon from './toolbar-icon';
+
+type TWorkspaceGroup = {
+    has_redo_stack: boolean;
+    has_undo_stack: boolean;
+    onResetClick: () => void;
+    onSortClick: () => void;
+    onUndoClick: (param?: boolean) => void;
+    onZoomInOutClick: (param?: boolean) => void;
+    toggleLoadModal: () => void;
+    toggleSaveModal: () => void;
+};
+
+const WorkspaceGroup = ({
+    has_redo_stack,
+    has_undo_stack,
+    onResetClick,
+    onSortClick,
+    onUndoClick,
+    onZoomInOutClick,
+    toggleLoadModal,
+    toggleSaveModal,
+}: TWorkspaceGroup) => (
+    <div className='toolbar__group toolbar__group-btn'>
+        <ToolbarIcon
```

Comment:
@prince-deriv i suggest that you consider mapping these multiple < ToolbarIcon />s based on the varying data we pass into them in order to reduce code duplication :)

Reply:
This was not feasible since the menu is divided into 2 parts